web/tendrl/multiuser_cluster_import.rst

```diff
@@ -1,2 +1,55 @@
-multiuser_cluster_import
-*******************************************************************************
\ No newline at end of file
+Multiuser - Cluster import
+****************************
+
+:authors:
+          - mkudlej@redhat.com
+:caseposneg: positive
+
+
+Description
+============
+
+Try to import cluster twice. Testcase is tested with one (or two users) and
+with 2 different browser sessions (two different session IDs). Goal is to try to import
+cluster from *Import wizard*. User(s) should have permission to import cluster.
+
+Setup
+======
+
+#. Prepare environment as described in :doc:`/configurations/multiuser_initial_setup`.
+
+#. Prepare cluster ready for import.
+
+Test Steps
+===========
+
+.. test_action::
+   :step:
+       In both browsers click on *Import Cluster* button.
+   :result:
+       In both browsers *Cluster Import* wizard starts.
+
+.. test_action::
+   :step:
+       In first browser click on *Import* button.
+   :result:
+       There is page for new task opened with message: *Import Cluster Task Submitted*
+
+.. test_action::
+   :step:
+       In first browsers click on *View Task Progress*.
+   :result:
+       Task in first browser should pass.
+
+.. test_action::
+   :step:
+       If import cluster task pass in second browser click on *Import* button.
```

Comment:
Maybe it's just my bad english, but I feel that the word order is bad.
I would rephrase it to something like:
> If import cluster task pass click on *Import* button in second browser.

Or
> Click on *Import* button in second browser if import cluster task pass (in the first one).

Hmm, probably even better will be to not use 'if import pass' but instead use 'when import pass'.